src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,24 @@ export function registerCommands(): void {
 		});
 	});
 }
+
+namespace GlobalQuickOpenCommand {
```

Comment:
@cristianhosu while this adds the command, it no longer shows up in the F1 command palette. @jrieken didn't we add support to contribute commands to the F1 list recently?